api/lock.json

```diff
@@ -0,0 +1,157 @@
+{
```

Comment:
The case of the file name should match that of the API, so this ought to be `Lock.json`